base/files/file_enumerator_unittest.cc

```diff
@@ -520,7 +520,13 @@ TEST(FileEnumerator, GetInfoRecursive) {
   // all the files.
   for (TestDirectory& dir : directories) {
     const FilePath dir_path = temp_dir.GetPath().Append(dir.name);
+#if defined(STARBOARD)
+#ifdef _WIN32
```

Comment:
Disable directory related tests for Windows here.
Should add a TODO here.
base::File is not fully migrated for Windows, as SBFileOpen use 2 different apis for directory open and file open. 

https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/shared/win32/file_open.cc;l=33?q=sbfileopen&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2FCOBALT

Reply:
I this gated on the other change from Madhu?

Reply:
Yes, base::File in base/files/file_starboard.cc will not be fully migrated until we support directory open for Windows.
In particular, we need to add directory open in void File::DoInitialize(const FilePath& path, uint32_t flags).